### DISCUSSION TITLE
Technical Discussion

### DISCUSSION THREAD

**Discussion by MuhammadKhalilzadeh** [verifywise-ai/verifywise#93] [General]
Here, we share our technical issues, solutions, and ideas.

**Comment by Omkar2101** [comment]
![console](https://github.com/user-attachments/assets/456fc12e-0c1a-45dd-9b82-fe5c10604b9e)
![profile](https://github.com/user-attachments/assets/11fd9d8d-d841-4396-8099-8f3814855fa0)
As shown in these images, the input fields do not display what the user is typing, and the console is logging an object when I fill in the input details

**Comment by gorkem-bwl** [comment]
I think @melpsh is still working on this page, so let's wait to hear from her.

**Reply by Omkar2101** [reply]
ok thanks for your reply
